src/dom-to-image-more.js

```diff
@@ -1193,20 +1193,66 @@
 
         function ensureSandboxWindow() {
             if (!sandbox) {
```

Comment:
[nit] Judging from block depth, this seems like a Yoda conditional and we can do this instead

```
            if (sandbox) {
                return sandbox.contentWindow;
            }

```